### PR TITLE
Adding cmlgenerator

### PR DESCRIPTION
**Pull request by mjw99**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by mdd31** [comment]
Happy to be added as a maintainer. 

The run_test.py needs to be created and added with calls to run the unit test suite crated with unittest.

**Comment by mjw99** [comment]
@conda-forge/help-python This PR is ready to be reviewed.

File: recipes/cmlgenerator/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+    - setuptools
+    - setuptools_scm
+  run:
+    - python >=3.6
+    - numpy
```
Comment (dopplershift) [inline recipes/cmlgenerator/meta.yaml:28]: Upstream also lists `lxml`. Any reason this doesn't?

Comment (mjw99) [reply recipes/cmlgenerator/meta.yaml:28]: Thanks for looking at this. Yes, I think you're correct. I will add this in. Many thanks for the spotting this.

**Comment by mjw99** [comment]
@conda-forge/help-python This PR is ready to be reviewed.

**Review by dopplershift — APPROVED** [review]

**Pull request merged**